plugins/3MFReader/ThreeMFWorkspaceReader.py

```diff
@@ -1133,7 +1133,15 @@ def _applyUserChanges(self, global_stack, extruder_stack_dict):
                         extruder_stack.userChanges.setProperty(key, "value", value)
                 if parser is not None:
                     for key, value in parser["values"].items():
-                        extruder_stack.userChanges.setProperty(key, "value", value)
+                        value_not_in_missing_package = True
+                        if "PLUGIN" in value:
```

Comment:
Maybe be more specific ? value.startwith("PLUGIN::")

You could also use a RegExp, it is a bit overkill for the test, but that allows you to extract the plugin name and version at the same time, so that the test below would be more simple.

These are just suggestions, feel free to reject them.

Reply:
I find it a bit complicated to use RegExp here. Do you have a suggestion?

Reply:
In this case it is not really helpful. However you could still use `value.startwith("PLUGIN::")` which will be much faster in case the `PLUGIN` word is not in the string (which is very likely). That implies it is always supposed to be at the beginning though.

Reply:
I did that in the new commit :)

Reply:
Yeah I just saw that :laughing: :+1: